commit/merkleroot/outcome.go

```diff
@@ -153,10 +167,15 @@ func buildReport(
 		outcomeType = ReportEmpty
 	}
 
+	if len(roots) > 0 && rmnEnabled && q.RMNSignatures == nil {
+		lggr.Errorw("RMN signatures are nil while RMN is enabled. Returning an empty outcome")
+		return Outcome{}
+	}
```

Comment:
Should this also return an error? Maybe include it in the verify function so that we fail prior to building a report.